adapters/dmx/dmx.go

```diff
@@ -221,21 +222,40 @@ func (adapter *DmxAdapter) MakeBids(request *openrtb.BidRequest, externalRequest
 }
 
 func fetchParams(params dmxExt, inst openrtb.Imp, ins openrtb.Imp, imps []openrtb.Imp, banner *openrtb.Banner, video *openrtb.Video, intVal int8) []openrtb.Imp {
+
 	if params.Bidder.TagId != "" {
 		ins = openrtb.Imp{
-			ID:     inst.ID,
-			TagID:  params.Bidder.TagId,
-			Ext:    inst.Ext,
-			Secure: &intVal,
+			ID:                inst.ID,
+			TagID:             params.Bidder.TagId,
```

Comment:
Wouldn't it be easier to just create a copy of `inst` as `ins` and then manipulate the fields that you need to manipulate? And you can do that outside of these two if conditions as everything else except the `tagID` value is the same in either case.

Reply:
@mansinahar Added test json for imp other expected value for banner and video